x-pack/test/spaces_api_integration/common/suites/resolve_copy_to_space_conflicts.ts

```diff
@@ -308,15 +316,15 @@ export function resolveCopyToSpaceConflictsSuite(
   ) => (): ResolveCopyToSpaceMultiNamespaceTest[] => {
     // the status code of the HTTP response differs depending on the error type
     // a 403 error actually comes back as an HTTP 200 response
-    const statusCode = outcome === 'noAccess' ? 404 : 200;
+    const statusCode = outcome === 'noAccess' ? 403 : 200;
     const type = 'sharedtype';
     const exactMatchId = 'all_spaces';
     const inexactMatchId = `conflict_1_${spaceId}`;
     const ambiguousConflictId = `conflict_2_${spaceId}`;
 
     const createRetries = (overwriteRetry: Record<string, any>) => ({ space_2: [overwriteRetry] });
     const getResult = (response: TestResponse) => (response.body as CopyResponse).space_2;
-    const expectForbiddenResponse = (response: TestResponse) => {
+    const expectForbiddenESResponse = (response: TestResponse) => {
```

Comment:
Nit: it's not intuitive, but this forbidden response isn't coming from Elasticsearch, it's coming from Kibana. Maybe it would be better to change this function name back to `expectForbiddenResponse` (since this same name is used many other places in the API integration test suites), and change the other function name to something like `expectNoAccessForbiddenResponse`.

Reply:
Fixed